playpen/mongoengine/dockerimage/rhel-pulp.repo

```diff
@@ -4,6 +4,15 @@
 [pulp-v2-stable]
 name=Pulp 2.8 Stable Builds
 baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/stable/2.8/7Server/$basearch/
+enabled=0
+skip_if_unavailable=0
+gpgcheck=1
+gpgkey=https://repos.fedorapeople.org/repos/pulp/pulp/GPG-RPM-KEY-pulp-2
+
+# Version 2.8 Beta Builds
+[pulp-v2-beta]
+name=Pulp 2.8 Beta Builds
+baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/beta/2.8/7Server/$basearch/
```

Comment:
After looking into this more, I think I know the answer to [my question](https://github.com/pulp/pulp/pull/2556/files/a4c3bcfa5c8da4bffa4bd7fc6a26eab02a533337#r61090071). The docker environment doesn't bundle or include the script in any way specifically. The value of rebuilding the docker update is that the test environment where a user runs a script will contain the latest model definitions and migrations. Given that as long as docker rebuilds when we GA a pulp release then I think we already have what we want. If ^ is true we should close this PR, and I regret asking you to update this when it wasn't needed.

Does this make sense to you?